.github/workflows/build_test.yml

```diff
@@ -36,8 +36,13 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+      - name: Download DeePMD-kit
+        run: |
+          mkdir -p ${{ runner.temp }}
+          wget https://github.com/deepmodeling/deepmd-kit/releases/download/v2.2.0.b0/libdeepmd_c.tar.gz -O ${{ runner.temp }}/libdeepmd_c.tar.gz --no-check-certificate
+          tar vxzf ${{ runner.temp }}/libdeepmd_c.tar.gz -C ${{ runner.temp }}
       - name: Build
         run: |
           git config --global --add safe.directory `pwd`
-          cmake -B build -DBUILD_TESTING=ON ${{ matrix.build_args }}
+          cmake -B build -DBUILD_TESTING=ON ${{ matrix.build_args }} -DDeePMD_DIR=${{ runner.temp }}/libdeepmd_c
```

Comment:
There is no deepmd building test yet. Maybe you should adding a test first.

Reply:
I don't quite understand the current implementation. Where is `DeepPot.init` called?

Reply:
I only see `DeepPot.compute` is called below:

https://github.com/deepmodeling/abacus-develop/blob/af20f70cccfecfe3bcf5211eae5d012b4cd37c7e/source/module_esolver/esolver_dp.cpp#L50

Reply:
> I don't quite understand the current implementation. Where is `DeepPot.init` called?

Hi @YuLiu98, would you take a look on this?

Reply:
> > I don't quite understand the current implementation. Where is `DeepPot.init` called?
> 
> Hi @YuLiu98, would you take a look on this?

`DeepPot.init` is called here: 
https://github.com/deepmodeling/abacus-develop/blob/af20f70cccfecfe3bcf5211eae5d012b4cd37c7e/source/module_esolver/esolver_dp.h#L16

Reply:
Hi @njzjz @caic99, is this PR complete? Or still missing the corresponding test?

Reply:
> Hi @njzjz @caic99, is this PR complete? Or still missing the corresponding test?

@YuLiu98 Yes, currently there is no building test for deepmd add-on. Maybe you can submit a new PR on it, and please check with Prof Chen first. @mohanchen 

@njzjz I suggest not introducing building tests in this PR, for the implementation does not conform to the current test system. What do you think?

Reply:
I feel fine to revert it.

Reply:
Done.